internal/health_checks/health_check_error.go

```diff
@@ -0,0 +1,120 @@
+		IsFatal:      true,
+	}
+	MON_API_CONN_ERR = HealthCheckError{
+		Code:         "MON_API_CONN_ERR",
+		Class:        "CONNECTION",
+		Message:      "Request to Monitoring API failed.",
+		Action:       "Check your internet connection.",
+		ResourceLink: "https://cloud.google.com/logging/docs/agent/ops-agent/troubleshooting",
+		IsFatal:      true,
+	}
+	LOG_API_SCOPE_ERR = HealthCheckError{
+		Code:         "LOG_API_SCOPE_ERR",
+		Class:        "PERMISSION",
+		Message:      "VM has not enough access scopes for the Logging API.",
+		Action:       "Add the https://www.googleapis.com/auth/logging.write scope to the GCP VM.",
+		ResourceLink: "https://cloud.google.com/logging/docs/agent/ops-agent/troubleshooting#logging-module-logs",
+		IsFatal:      true,
+	}
+	MON_API_SCOPE_ERR = HealthCheckError{
+		Code:         "MON_API_SCOPE_ERR",
+		Class:        "PERMISSION",
+		Message:      "VM has not enough access scopes for the Monitoring API.",
+		Action:       "Add the https://www.googleapis.com/auth/monitoring.write scope to the GCP VM.",
+		ResourceLink: "https://cloud.google.com/logging/docs/agent/ops-agent/troubleshooting",
+		IsFatal:      true,
+	}
+	LOG_API_PERMISSION_ERR = HealthCheckError{
+		Code:         "LOG_API_PERMISSION_ERR",
+		Class:        "PERMISSION",
+		Message:      "Service account misssing permissions for the Logging API.",
+		Action:       "Add the logging.writer role to the GCP service account.",
+		ResourceLink: "https://cloud.google.com/logging/docs/agent/ops-agent/troubleshooting#logging-module-logs",
+		IsFatal:      true,
+	}
+	MON_API_PERMISSION_ERR = HealthCheckError{
+		Code:         "MON_API_PERMISSION_ERR",
+		Class:        "PERMISSION",
+		Message:      "Service account misssing permissions for the Monitoring API.",
+		Action:       "Add the monitoring.writer role to the GCP service account.",
+		ResourceLink: "https://cloud.google.com/logging/docs/agent/ops-agent/troubleshooting",
+		IsFatal:      true,
+	}
+	LOG_API_DISABLED_ERR = HealthCheckError{
+		Code:         "LOG_API_DISABLED_ERR",
+		Class:        "API",
+		Message:      "The Logging API is disabled in the current GCP project.",
+		Action:       "Enable Logging API in the current GCP project.",
+		ResourceLink: "https://cloud.google.com/logging/docs/agent/ops-agent/troubleshooting#logging-module-logs",
+		IsFatal:      true,
+	}
+	MON_API_DISABLED_ERR = HealthCheckError{
+		Code:         "MON_API_DISABLED_ERR",
+		Class:        "API",
+		Message:      "The Monitoring API is disabled in the current GCP project.",
+		Action:       "Enable Monitoring API in the current GCP project.",
+		ResourceLink: "https://cloud.google.com/logging/docs/agent/ops-agent/troubleshooting",
+		IsFatal:      true,
+	}
+	HC_FAILURE_ERR = HealthCheckError{
+		Code:         "HC_FAILURE_ERR",
+		Class:        "GENERIC",
+		Message:      "The Health Check failed.",
+		Action:       "No suggested action.",
```

Comment:
```suggestion
		Action:       "...<something along the lines of "open a case with Google Cloud support">...",
```

Reply:
Addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1051/commits/d2845edbb605e75924e9a00dacde743bd776558b .